test/SparqlExpressionTest.cpp

```diff
@@ -415,12 +415,12 @@ TEST(SparqlExpression, dateOperators) {
                    std::optional<int> expectedYear,
                    std::optional<int> expectedMonth,
                    std::optional<int> expectedDay,
-                   std::optional<int> expectedHours,
-                   std::optional<int> expectedMinutes,
-                   std::optional<double> expectedSeconds,
+                   std::optional<int> expectedHours = std::nullopt,
+                   std::optional<int> expectedMinutes = std::nullopt,
```

Comment:
You can also do the same for `Month` and `Day` to make the tests for `gYear` and `gYearMonth` below also a little shorter.

Reply:
Done.